dev/testing/en/getting-started.md

```diff
@@ -6,6 +6,22 @@ title: Getting Started - Testing - OJS/OMP
 
 We use [Cypress](https://www.cypress.io/) and [PHPUnit](https://phpunit.de/) to test our applications. Before you can run the tests, you will need to [clone and configure](/dev/documentation/en/getting-started) the application's repository so that you have a working installation of the application on your local environment.
 
+## Requirements
+
+In main branch or OJS Versions released after 3.5, you will need the following  email setup to locally  run the cypress tests.
```

Comment:
@withanage does this apply only to OJS testing or to all applications?

Reply:
@kaitlinnewson currenlty only for OJS